README.md

```diff
@@ -9,25 +9,40 @@ The following steps will install the `tigerbeetle-node` module to your current w
 * NodeJS >= `14.0.0`. _(If the correct version is not installed, an installation error will occur)_ 
 
 > Your operating system should be Linux (kernel >= v5.6) or macOS. Windows support is not yet available but is in the works.
-     
+
 ### YARN Package Manager
 
-```sh
-# Run the following from this directory
-yarn add
+```shell
+# Run the following from this directory:
+yarn postinstall #Download NodeJS Headers, Install Zig, Build TigerBeetle
+yarn && yarn build && yarn add
```

Comment:
@koekiebox thanks for updating this part.
I can't remember why the `Run the following from this directory` is there. The intention of this part is to show how you would install the `tigerbeetle-node` package in your own project. You would normally just go `yarn add tigerbeetle-node` or `npm install tigerbeetle-node`. Similar to https://github.com/koajs/koa.

Are you trying to show instructions to get started for development?

Reply:
@DonChangfoot Ha, I think I am the guilty party, not knowing exactly how `yarn/npm` worked at the time. 
I have updated the `README.md` to split between running the test cases and making use of the `tigerbeetle-node` client in your own NodeJS project. :smile: